packages/runner-sdk/models.d.ts

```diff
@@ -358,6 +358,9 @@ export declare class NangoAction {
      * Only use that method when you want to access resources that are unrelated to the current connection/provider.
      */
     uncontrolledFetch(options: { url: URL; method?: HTTP_METHOD; headers?: Record<string, string> | undefined; body?: string | null }): Promise<Response>;
+    tryAcquireLock(props: { key: string; ttlMs: number }): Promise<boolean>;
+    releaseLock(props: { key: string }): Promise<boolean>;
+    releaseAllLocks(): Promise<void>;
```

Comment:
we don't need to expose `releaseAllLock` but why not. wdyt?

Reply:
People will use and abuse anything we expose. I wouldn’t expose it personally but if we did would give it a clearer name